src/components/input/math-input.js

```diff
@@ -824,7 +824,10 @@ class MathInput extends React.Component {
         // keyboard appear. It should only require one finger, which is how iOS works.
         // TODO(diedra): Fix the bug that is causing Android to require a two finger tap
         // to the open the keyboard, and then remove the second half of this label.
-        const ariaLabel = `{i18n._("Math input box") i18n._("Tap with one or two fingers to open keyboard")}`;
```

Comment:
Huh, what was this even doing before?

Reply:
Although, confusingly, fixing this at the same time as updating the version isn't correct! Since this fix isn't in 0.2.5, right? You should move this to 0.2.6

Reply:
I dunno, probably none of this matters though :-P

Reply:
Yeah, that's a good point about the fix not being in 0.2.5.  This should've been two separate commits.